docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -581,3 +598,85 @@ If processing of any rule fails, a partial error outputs the ID and/or name of t
 --------------------------------------------------
 
 <1> dev:[] `execution_summary` is under development and its schema may change.
+
+*Example 3, Dry run*
+
+The following request validates `add_index_patterns` action to the rules, which IDs included in the payload.
+Because, it's in dry ryn mode: no changes will be saved for these rules.
+
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/rules/_bulk_action?dry_run=true
+{
+    "action": "edit",
+    "edit": [
+        {
+            "value": [
+                "test-*"
+            ],
+            "type": "add_index_patterns"
+        }
+    ],
+    "ids": ["81aa0480-06af-11ed-94fb-dd1a0597d8d2", "dc015d10-0831-11ed-ac8b-05a222bd8d4a", "de8f5af0-0831-11ed-ac8b-05a222bd8d4a"]
+}
+--------------------------------------------------
+
+[discrete]
+===== Response code
+
+`500`::
+    Indicates partial bulk action failure.
```

Comment:
```suggestion
    Indicates a partial bulk action failure.
```

Reply:
I don't think  indefinite article is required here. cc @nastasha-solomon

Reply:
Both versions of this work! That said, I have noticed that indefinite articles are commonly used throughout this topic (see [here](https://www.elastic.co/guide/en/security/master/bulk-actions-rules-api.html#_response_code_9) for an example) so I'd include it for consistency if nothing else.